packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -124,8 +124,6 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, () => {
 	const nodeHelpers = useNodeHelpers();
 	const usersStore = useUsersStore();
 
-	const version = settingsStore.partialExecutionVersion;
```

Comment:
Sorry @despairblue I have overlooked it.
If you use it here, at the store initialization function, it should be
`const version = computed(() => settingsStore.partialExecutionVersion);`

And use it like ``/workflows/${startRunData.workflowData.id}/run?partialExecutionVersion=${version.value}`` in the `runWorkflow` function